### ISSUE TITLE
Non-frontal face detection

### ISSUE THREAD

**Issue by Santhanalakshmimano** [fyr91/face_detection#5]
I would like to use ultra light detector. Can it detect non-frontal faces or faces with mask as well Sir ? Could you please let me know

**Comment by fyr91** [comment]
Yes, it will be able to detect face from different angles and with mask

Santhanalakshmimano ***@***.***> 于2021年3月17日周三 下午10:24写道：

**Comment by Santhanalakshmimano** [comment]
Thank you so much Sir
